requirements/requirements-dev.txt

```diff
@@ -1,24 +1,22 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
+# This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --extra=quality --output-file=requirements/requirements-dev.txt pyproject.toml
+#    pip-compile --extra=quality --output-file=/Users/iholmes/safejax/requirements/requirements-dev.txt /Users/iholmes/safejax/requirements/../pyproject.toml
```

Comment:
This is what I meant above, we should avoid generating absolute paths there, that also include personal information i.e. `Users/iholme/safejax`

Changing the `REQPATH` above would do the work, also assuming the script will always be run within the root directory of the repository. So on, the generated file should look like:

```suggestion
#    pip-compile --extra=quality --output-file=requirements/requirements-dev.txt pyproject.toml
```